disasm/isa_parser.cc

```diff
@@ -359,11 +368,11 @@ isa_parser_t::isa_parser_t(const char* str, const char *priv)
     bad_isa_string(str, "'Zcf/Zcd/Zcb/Zcmp/Zcmt' extensions require 'Zca' extension");
   }
 
-  if (extension_table[EXT_ZACAS] && !extension_table['A']) {
+  if (extension_table[EXT_ZACAS] && !extension_table['A'] && !extension_table[EXT_ZAAMO]) {
     bad_isa_string(str, "'Zacas' extension requires 'A' extension");
```

Comment:
Can you update the error message to match the new logic?

Reply:
updated.